### PR TITLE
Add textfile parser

### PR DESCRIPTION
**Pull request by 3ter**
Every new function should be visible on one page, I tried to keep them in small readable chunks.

The parser reads a markdown file and executes its commands in two parts:
- Non-interactive part, just press enter to read and listen
- Interactive part, enter commands and see the responses

The only ambience implemented right now is sound (single file, no track map yet, but not too much work either I suppose).

### CODE REVIEW

**Comment by damaskino** [comment]
Looks good, I've tested the start scene with the commands described in the sceneOneBeach.md and I couldn't get it to crash through the commands, I've tried inputting commands earlier than they were expected and just nothing happened, so it seems robust in that regard. 
Only times it crashes seems to be related to the audio files, which can be resolved by restarting the game, but I will open a separate issue for that. 
I fixed a small typo that I noticed, other than that, it can be merged to master from my end.

File: internal/scene/sceneOneBeach.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+`(Go west) > gone_west`
+
+# gone_west
+You leave the sand on the ground and reach grass. A lighthouse is built here. You can see light on top. It’s not very bright. You wonder if someone lives here.
+
+`(enter lighthouse) > lighthouse_enter`
+
+# lighthouse_enter
+You enter the lighthouse. The door was not locked. The inside looks abandoned, as if the last time a human inhibited this room was over a decade ago.
```
Comment (damaskino) [inline internal/scene/sceneOneBeach.md]: change "inhibited" to "inhabited"

File: internal/scene/sceneOneBeach.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+`(Inspect reflection) > get_compass`
+
+# get_compass
+You walk closer to whatever it is that caught your eye. It was glass that reflected sunlight into your eyes. Glass that belonged to a little device. A compass.
+
+`(Pick up compass) > got_compass`
+
+# got_compass
+`[Audio: Harp.ogg]`
```
Comment (damaskino) [inline internal/scene/sceneOneBeach.md:18]: The Audio change to the song in here overwrites the waves, which kind of took me out of the moment. Would be nicer to have the waves continuing in the background while we're in this scene, but we can add that in a future update and merge this first

**Review by damaskino — APPROVED** [review]
Getting to grips with this function myself, but looks good so far, only 1 typo and a feature that we can add in a later update.
My name is damaskino and I approve this pull request ;)

File: internal/scene/sceneOneBeach.md (hunk 2 above)
Comment (3ter) [reply internal/scene/sceneOneBeach.md:18]: I found the harpies rather intriguing in that moment ;-). But as it's only a placeholder as I understood it, I've just wanted to check if the audio actually switched over to a new track.

**Pull request merged**